### DISCUSSION TITLE
[loki] explore: add sort by time button

### DISCUSSION THREAD

**Discussion by liguozhong** [grafana/grafana#58735] [Observability]
![image](https://user-images.githubusercontent.com/9583245/201871682-13329820-bca6-4057-afbd-834b42013b60.png)

add one button (sort by time) when search by uuid or traceid from micro service ，we need analyse log sorted by time.

thanks all

`{log_type="system_log", debug="true"}|="1ac5d591-43a9-4349-a678-c91f7cc0b48d"
`
`Limit: 1000 (473 returned) Total bytes processed: 14 GB
`
When explore finds 473 out-of-order logs printed by 14 microservices with the same traceid, it is difficult for users to analyze these logs.

If it is sorted in chronological order, the user will easily understand how the system behind this traceid works

**Comment by liguozhong** [comment]
help wanted

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
